SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Microsoft.AppCenter.WindowsDesktop.csproj

```diff
@@ -18,6 +18,7 @@
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
     <PackageReference Include="SQLitePCLRaw.bundle_green" Version="2.0.2" />
+    <PackageReference Include="System.Net.Http" Version="4.3.4" />
```

Comment:
There was no warning for this sub-project (SDK part). Why did you add extra dependency here?

Reply:
Those are the packages \ sub-projects using System.Net.Http; I think fixing all the related places would be a good call, rather than those which were detected.